cmd/loadTests.go

```diff
@@ -545,6 +545,22 @@ func StageCleanup(users []loadtestUtils.User) {
 
 }
 
+// StageCleanup at the end of the load-tests doesn't delete the CDQs of the users, therefore
+// We should explicitely delete it per user at the early stage of the userJourneyThread fucntion
+// To make things more robust we shall in creation reverse order, delete the user components and then the CDQs of the user
+func StageCleanupSingle(user loadtestUtils.User, framework *framework.Framework) {
```

Comment:
Should this function be somehow connected with `StageCleanup`? Just having one function that would delete all entities that need to be deleted.